docs/advanced/symbol-server-compatibility.md

```diff
@@ -75,6 +75,11 @@ Specifically, the code and debug identifiers are defined as follows:
   bidirectionally from the UUID.
 - **Debug ID:** The same UUID, amended by a `0` for age.
 
+**WASM**:
+
+- **Code ID:** The bytes as specified in the `build_id` custom section.
+- **Debug ID:** The same as code ID but truncated to 16 bytes.
```

Comment:
any reason to want to truncate this?  Or is this already established?

Reply:
All debug IDs are 16+4 bytes. We generally try not to use the 4 byte age field due to the bad support.